### DISCUSSION TITLE
關於 branch 的規劃方式？

### DISCUSSION THREAD

**Discussion by 4hfly** [4hfly/Associated_Learning#9] [General]
目前有三個：
  * main
    > 不可直接 commit，只能透過 PR 更新。
  * dev
    > 開發用。
  * lm 
    > 最近這邊更新應該會比較頻繁？

這樣如何？

**Comment by Hibb-bb** [comment]
ok, lm其實也快用完了 再來應該就是nmt的部分要花比較多時間。
大致上nmt需要的東西我寫一下

1. bi-gru nmt (可以用之前參考的那個)
2. bi-gru AL nmt
3. evaluation(BLEU, ROUGE)
